Author: Paul Sladen <debian@paul.sladen.org>
Date: Mon Jul 4 10:56:34 2016 +0200
Description: Remove JCompoBox<String> from FrameColorMap for compatibility
Index: aladin/cds/aladin/FrameColorMap.java
===================================================================
--- aladin.orig/cds/aladin/FrameColorMap.java	2016-12-19 11:51:39.513746219 +0100
+++ aladin/cds/aladin/FrameColorMap.java	2016-12-19 12:00:42.945735067 +0100
@@ -99,10 +99,10 @@
    private JPanel p=null;                          // Le panel courant
    private JTextField pixelCutMinField;            // Champ de saisie pour le pixelCutMax
    private JTextField pixelCutMaxField;            // Champ de saisie pour le pixelCutMax
-   private JComboBox<String> comboCM;              // Pour le choix de la colormap
+   private JComboBox comboCM;              // Pour le choix de la colormap
 
    private JButton localCutButton, getAllButton, resetAllButton, resetDistribButton, resetCMButton, applyOnAll;
-   private JComboBox<String> cmCombo, fctCombo;
+   private JComboBox cmCombo, fctCombo;
    private JCheckBox reverseCb;
    private JRadioButton rPreview,rFull;
 
@@ -490,7 +490,7 @@
 
       JButton b;
       JLabel l;
-      JComboBox<String> cb;
+      JComboBox cb;
       JCheckBox cc;
 
       l = new JLabel( Util.fold(CMMETHOD,30,true) );
@@ -737,16 +737,16 @@
    public Dimension getMinimumSize() { return new Dimension(200,300); }
 
    /** Création d'un Combo des CM possibles */
-   protected static JComboBox<String> createComboCM() {
-      JComboBox<String> c = new JComboBox<String>();
+   protected static JComboBox createComboCM() {
+      JComboBox c = new JComboBox();
       for( String s : CanvasColorMap.getCMList() ) c.addItem(s);
       c.addItem(" -- ");
       return c;
    }
 
    /** Création d'un Combo des Fonctions de transferts possibles */
-   protected static JComboBox<String> createComboFct() {
-      JComboBox<String> c = new JComboBox<String>( PlanImage.TRANSFERTFCT );
+   protected static JComboBox createComboFct() {
+      JComboBox c = new JComboBox( PlanImage.TRANSFERTFCT );
       return c;
    }
 
@@ -907,7 +907,7 @@
    }
 
    /** Changement de la color map */
-   private void changeCM(JComboBox<String> c) {
+   private void changeCM(JComboBox c) {
       int n = c.getSelectedIndex();
       if( n==pimg.typeCM ) return;
       pimg.typeCM = n;
